nulib/dc-nextjs
# Updates related to Lighthouse feedback

## What does this do?
Makes some SEO improvements to the page based on feedback from Lighthouse.  Most of the improvement suggestions were related to dependent packages.  Within this code base, the 2 primary updates were...

1. Pulling in `next/font` and using local fonts instead of grabbing from NU CDN.
2. Pulling in the Bundle Analyzer package to help identify JS weight and bundle size.

## To Test
Feel free to run the Bundle Analyzer, Lighthouse in general, and make sure there are no odd font-related display issues.
